README.md

```diff
@@ -1,98 +1,168 @@
+```
+
+#### Using a CLI command
+
+The following command generates a `backup.tar` file with two blocks of 512 zero bytes
 
 ```bash
-drwxr-xr-x  0 root   root        0 Sep 21 14:53 volumes/mock-avs-second/1821030e0e15cc096c6c3a13c936baad1b8a4a98f0cbaf2b25cd4642203093fd
--rw-r--r--  0 root   root      653 Sep 22 11:46 volumes/mock-avs-second/1821030e0e15cc096c6c3a13c936baad1b8a4a98f0cbaf2b25cd4642203093fd/cli.go
--rw-r--r--  0 root   root     1929 Sep 22 14:52 volumes/mock-avs-second/95e0a42d9d6b5d82bdb3752f4d31f3fe7d0150c6b512bc094985b1a2b24b192b
--rw-------  0 root   root      203 Sep 22 14:52 volumes/mock-avs-second/volumes-data.yml
+dd if=/dev/zero of=backup.tar bs=512 count=2
+```
+
+```text
+2+0 records in
+2+0 records out
+1024 bytes (1.0 kB, 1.0 KiB) copied, 0.00199071 s, 514 kB/s
 ```
 
-Notice the presence of the `volumes-data.yml` file. This file contains metadata about the list of volumes in the prefix path. This file is used by the restore process to restore the volumes in the same path. The content of this file is:
+### Passing the backup `tar` file
 
-```yaml
-- id: 1821030e0e15cc096c6c3a13c936baad1b8a4a98f0cbaf2b25cd4642203093fd
-  type: dir
-  target: /cli
-- id: 95e0a42d9d6b5d82bdb3752f4d31f3fe7d0150c6b512bc094985b1a2b24b192b
-  type: file
-  target: /README.md
+The snapshotter process requires the backup tar file to be located at the `/backup.tar` path inside the cotainer. Therefore, the proper way to pass the configuration is by mounting the following volume:
+
+```text
+--volume <path-to-backup-tar>:/backup.tar
 ```
+
+Replace `<path-to-backup-tar>` with absolute path to the configuration file on the host machine.
```

Comment:
Configuration file or backup tar. Check also the last sentence of the above paragraph.

Reply:
Well noticed. Fixed here https://github.com/NethermindEth/docker-volumes-snapshotter/pull/1/commits/89ee98b0715b976c409e5f52a1a8b5d9ae959511